build.gradle

```diff
@@ -71,7 +71,7 @@ configurations {
     all {
         resolutionStrategy {
             // pravega/pravega/issues/2464
-            force 'io.netty:netty-tcnative-boringssl-static:2.0.8.Final'
+            force "io.netty:netty-tcnative-boringssl-static:" + nettyBoringSSLVersion
```

Comment:
Do we still need to continue to force a specific version of BoringSSL through an explicit resolution strategy? If the regular dependency declaration works now, it'd be good to use that, so that this one doesn't stand out.

Reply:
Yes we don't need to keep this since the version is upgraded in Pravega. Will remove it.